pkg/acceptance/helpers/test_client.go

```diff
@@ -51,3 +69,19 @@ type TestClientContext struct {
 	schema    string
 	warehouse string
 }
+
+func (c *TestClientContext) databaseId() sdk.AccountObjectIdentifier {
+	return sdk.NewAccountObjectIdentifier(c.database)
+}
+
+func (c *TestClientContext) schemaId() sdk.DatabaseObjectIdentifier {
+	return sdk.NewDatabaseObjectIdentifier(c.database, c.schema)
+}
+
+func (c *TestClientContext) warehouseId() sdk.AccountObjectIdentifier {
+	return sdk.NewAccountObjectIdentifier(c.warehouse)
+}
+
+func (c *TestClientContext) newSchemaObjectIdentifier(name string) sdk.SchemaObjectIdentifier {
```

Comment:
There is currently no place in which it would be utilized without changes (we can change the signature of two methods to leverage it). So, for now, I think there is no need for it.